packages/components/src/components/Alert/Alert.tsx

```diff
@@ -194,6 +162,52 @@ export const Alert: FC<AlertProps> = ({
     lineHeight: 30,
   }
 
+  const _header = () => {
+    const headerText = header ? (
+      <Text style={headerFont}>{header}</Text>
+    ) : description ? (
+      <Text style={descriptionFont}>{description}</Text>
+    ) : null
+
+    const a11yLabel = header
+      ? headerA11yLabel || header
+      : description
+        ? descriptionA11yLabel || description
+        : ''
+
+    /**
+     * Wrap header text and expand icon in Pressable if the Alert is expandable
+     * Otherwise wrap in View with accessibility props
+     */
+    if (expandable) {
+      return (
+        <Pressable
+          onPress={() => setExpanded(!expanded)}
+          accessibilityRole="tab"
+          accessibilityState={{ expanded }}
+          aria-label={a11yLabel}
+          style={{ flexDirection: 'row' }}>
+          <View style={{ flex: 1 }}>{headerText}</View>
+          <View style={iconViewStyle}>
+            <Spacer horizontal />
+            <Icon
```

Comment:
Maybe worth checking with Jessica by bumping the [Slack Q&A thread](https://adhoc.slack.com/archives/C05DQPHUVNZ/p1709747499865419) but based on Figma boxes for [expandable Alert examples](https://www.figma.com/file/Zzt8z60hCtdEzXx2GFWghH/branch/DlEO0NFO9TclCmAqQ8giCi/%F0%9F%93%90-Component-Library?type=design&node-id=258%3A2769&mode=design&t=Fxf6YQGYx7hTpdrL-1), it visually appears like the the chevron should also carve out space from the content so that text doesn't appear below the chevron when expanded and would wrap before being out that far.

If that is the case, I think this Icon needs to be moved below to be in the same View with `iconDisplay` below.

Reply:
[Added a Q&A in the thread](https://adhoc.slack.com/archives/C05DQPHUVNZ/p1713289982748569?thread_ts=1709747499.865419&cid=C05DQPHUVNZ) for this while adding another somewhat related Q&A.

Reply:
Added a spacer to the right of the content to achieve this margin when expandable. Considered using flexbox for this, but it would have required me separating the single Pressable into 2 Pressables for the title and chevron. Thought it'd be better for the screenreader to just have the one.

Reply:
I think we are going to want to unify this in some capacity with the logic below. Can check with Jessica, but playing with the component currently the touch target for expand/collapse is currently only the header text (and trailing space) + the chevron. It doesn't include the alert icon (info/check/etc.) or space that's visually part of the collapsed Alert but not close enough to the text/chevron, leaving a fairly large area to the left end and the border on top/right/bottom sides that doesn't toggle the expand/collapse behavior. 

Intuitively, I think tapping anywhere on the collapsed Alert should expand it--and similarly tapping anywhere along the top portion should collapse when expanded. Perhaps even further: when expanded tapping anywhere on the Alert that isn't an internal pressable (Buttons, children components with onPress) should also collapse it.

Basically: wondering if the entire Alert should be wrapped in the Pressable and we just have no onPress for non-expandable versions (and correspondingly hide the expand/collapse a11y stuff behind expandable).

Reply:
Confirmed with Jessica that the entire Alert should be pressable when it's collapsed. Will have to rework the structure so that header is separate from content.

Reply:
Alternatively, perhaps we [use `hitSlop`](https://reactnative.dev/docs/touchablewithoutfeedback#hitslop) with a Rect object to fill out the padding to be pressable? Doing it that way, we could also pull the Chevron out (to use flexbox like the alert icon) and ensure the entire "header" is pressable when open or closed. Not sure if that might get weird around font scaling with the hitSlop though.